### ISSUE TITLE
Feature Idea: Retrieve "Full Record and Cited References"

### ISSUE THREAD

**Issue by frajuegies** [vt-arc/wosr#5]
Hi,

thank you very much for your great package.

Do you think it is possible to retrieve and parse the "Full Record and Cited References" as an option like you find this on the WOS download screen?

![wos download](https://user-images.githubusercontent.com/3856556/46515929-61e5e880-c856-11e8-994f-198185553a71.PNG)

This would be great to enable network graphs...

Best regards,

Jürgen

**Comment by crew102** [comment]
Hi Jürgen, the "full record and cited references" format that you referenced is not served by the WoS API, but the API does offer cited references for a given publication in a different format. For example, here is the data that is provided for one cited reference:

![screen shot 2018-10-05 at 6 53 22 pm 2](https://user-images.githubusercontent.com/11220793/46563668-d3627d00-c8d0-11e8-9dbe-d0ef7ec9be33.jpg)

I'll consider adding a client to `wosr` that pulls this data. It's not high on my priority list, though, so I may not get around to it for awhile.

**Comment by frajuegies** [comment]
Thank you very much.... i will try to learn from your functions used in wosr.... and perhaps you are quicker than I 😊

Best regards 
Jürgen

**Comment by crew102** [comment]
Hi @frajuegies , this issue has been closed by commits 83ebd13e4f8f5c807fecf06af7c511aa2eed349d- 2b04459b6a950008e28cb0a57cdad7e89a6f3815. 

After you install the development version of `wosr`, you should be able to pull a publication's cited references like so:

``` r
library(wosr)
sid <- auth()
uts <- c("WOS:000362312600021", "WOS:000439855300030", "WOS:000294946900020")
pull_cited_refs(uts, sid)
#>                    ut    doc_id
#> 1 WOS:000362312600021  77452845
#> 2 WOS:000362312600021   4418415
#> 3 WOS:000362312600021 286622905
#> 4 WOS:000439855300030 558219860
#> 5 WOS:000294946900020 422906512
#> 6 WOS:000294946900020 422609377
#>                                                                                                title
#> 1                         Factors affecting time to adoption of dogs re-homed by a charity in the UK
#> 2                       Factors associated with the decision to surrender a pet to an animal shelter
#> 3 Characteristics of Relinquished Dogs and Their Owners at 14 Rehoming Centers in the United Kingdom
#> 4         Peak forelimb ground reaction forces experienced by dogs jumping from a simulated car boot
#> 5                                                                                               <NA>
#> 6                                                                                               <NA>
#>                                                  journal          author
#> 1                                         ANIMAL WELFARE      Diesel, G.
#> 2 JOURNAL OF THE AMERICAN VETERINARY MEDICAL ASSOCIATION      Miller, DD
#> 3              JOURNAL OF APPLIED ANIMAL WELFARE SCIENCE Diesel, Gillian
#> 4                                      VETERINARY RECORD   Pardey, David
#> 5                                                VET REC            <NA>
#> 6                                                VET REC            <NA>
#>   tot_cites year page volume
#> 1        20 2007  353     16
#> 2       108 1996  738    209
#> 3        28 2010   15     13
#> 4         1 2018  716    182
#> 5         1 2011  291    169
#> 6         2 2011  217    169
```

Created on 2018-10-17 by the [reprex package](https://reprex.tidyverse.org) (v0.2.0.9000).

**Comment by frajuegies** [comment]
Good morning,
thank you very much for the implementation....
Hopefully I can do a first test this weekend.
Best regards,
Jürgen
